hack/lib/strimzi.bash

```diff
@@ -134,11 +134,49 @@ EOF
       --from-literal=user="my-sasl-user"
 }
 
+function install_kafka_ui {
+  logger.info "Installing Kafka UI"
+  cat <<-EOF | oc apply -f -
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kafka-ui
+  namespace: kafka
+  labels:
+    app: kafka-ui
+spec:
+  containers:
+    - env:
+      - name: KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS
+        value: my-cluster-kafka-bootstrap.kafka.svc:9092
+      - name: KAFKA_CLUSTERS_0_NAME
+        value: my-cluster
+      image: docker.io/provectuslabs/kafka-ui
```

Comment:
will this get same "rate limit" issues we had sometime ago w/ maistra? Or is their accoount a "better" one? :see_no_evil:

Reply:
I guess we can't do much, when they publish to dockerhub... :-)

Reply:
We could probably mirror this image. Maybe I should push it to quay?

Reply:
Pushing to quay. Will update this PR when it's ready